### PR TITLE
[DO NOT MERGE] Office 365 usage reports CSV version to Beta

### PR DESCRIPTION
**Pull request by kszb**

### CODE REVIEW

File: api-reference/beta/api/reportroot_getemailactivitycounts.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# reportRoot: getEmailActivityCounts
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/api/reportroot_getemailactivitycounts.md]: remember the pattern for the title: binding/related entity: functionName function

File: api-reference/beta/api/reportroot_getemailactivityuserdetail.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "stream"
+} -->
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/octet-stream
+
+Report Refresh Date,User Principal Name,Display Name,Is Deleted,Deleted Date,Last Activity Date,Send Count,Receive Count,Read count,Assigned Products,Report Period
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/api/reportroot_getemailactivityuserdetail.md]: for consistency, should be Read Count

Comment (kszb) [reply api-reference/beta/api/reportroot_getemailactivityuserdetail.md]: Thanks for this detail review! Updated.

File: api-reference/beta/api/reportroot_getoffice365groupsactivitygroupcounts.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# reportRoot: getOffice365GroupsActivityGroupCounts
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Get the daily total number of groups and how many of them were active based on Email Conversations, Yammer Posts and SharePoint file activities.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/api/reportroot_getoffice365groupsactivitygroupcounts.md]: Update end of sentence to:
email conversations, Yammer posts, and SharePoint file activities.

File: api-reference/beta/api/reportroot_getonedriveusageaccountcounts.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+# reportRoot: getOneDriveUsageAccountCounts
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Get the trend in the number of active OneDrive for Business sites. Any site on which users viewed, modified, uploaded, downloaded, shared, synced files is considered an active site.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/api/reportroot_getonedriveusageaccountcounts.md]: please put an "or": shared, or synced

File: api-reference/beta/api/reportroot_getskypeforbusinessparticipantactivityminutecounts.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+# reportRoot: getSkypeForBusinessParticipantActivityMinuteCounts
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Get usage trends on the length in minutes and type of conference sessions that users from your organization participated in. Types of conference sessions include audio/video, and dial-in and dial-out - Microsoft.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/api/reportroot_getskypeforbusinessparticipantactivityminutecounts.md]: The metadata and the CSV columns listed don't include "dial-in and dial-out - Microsoft". Should it be included in the description or is it missing from the metadata/CSV?

File: api-reference/beta/api/reportroot_getyammeractivityusercounts.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
+# reportRoot: getYammerActivityUserCounts
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Get the trends on the number of unique users who post, read, and like Yammer messages.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/api/reportroot_getyammeractivityusercounts.md]: Please update to: posted, read, and liked

File: api-reference/beta/api/reportroot_getyammergroupsactivitycounts.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
+# reportRoot: getYammerGroupsActivityCounts
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Get the number Yammer messages posted, read, and liked in groups.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/api/reportroot_getyammergroupsactivitycounts.md]: Should be: number of Yammer

File: api-reference/beta/resources/office_365_groups_activity_reports.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# Office 365 groups activity reports
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/resources/office_365_groups_activity_reports.md]: Should be Office 365 Groups. Update where referenced in titles and descriptions.

File: api-reference/beta/resources/yammer_activity_reports.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+# Yammer activity reports
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+You can understand the level of engagement of your organization with Yammer by looking at the number of unique users using Yammer to post, like or read a message and the amount of activity generated across the organization.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/resources/yammer_activity_reports.md]: Recommendation:
You can understand the level of your organization's engagement with Yammer by how much activity is generated across the organization and the number of unique users who post, like, and read messages on Yammer.

File: api-reference/beta/resources/yammer_device_usage_reports.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+# Yammer device usage reports
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+The Yammer device usage reports give you information about which devices your users are using Yammer on. You can view the number of daily users by device type, and number of users by device type. You can view both over a selected time period. You can also view details per user.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/resources/yammer_device_usage_reports.md]: Recommendation for the first sentence:
The device usage reports for Yammer give you information about which devices your users utilize to engage on Yammer.
Recommendation for the remaining sentences:
You can view the number of users by device type over a selected time period and view details per user.

File: api-reference/beta/resources/skype_for_business_peer_to_peer_activity.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
+# Skype for Business peer-to-peer activity reports
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+You can get details on peer-to-peer activity across your organization. These details are very helpful when you investigating, planning, and making other business decisions for your organization.
```
Comment (ElizabethSamuel-MSFT) [inline api-reference/beta/resources/skype_for_business_peer_to_peer_activity.md]: Please add "are": when you are investigating

**Pull request merged**